src/screentovnc.h

```diff
@@ -79,6 +84,9 @@ enum Orientation {
 static rfbCursor *emptyMousePtr;
 static rfbCursor *pointerFingerPtr;
 static rfbCursor *pointerFingerTouchPtr;
+static cursor_info emptyMouseCursorInfo;
+static cursor_info pointerCursorInfo;
+static cursor_info pointerTouchCursorInfo;
```

Comment:
Wondering is there any need for keeping these after rfbMakeXCursor() calls unless the cursor keeps some references on them. Otherwise these could be perhaps just temporary variables in init_fingerPointers()?